nsdb-http/src/main/scala/io/radicalbit/nsdb/web/routes/CommandApi.scala

```diff
@@ -302,26 +353,25 @@ trait CommandApi {
       new ApiResponse(code = 500, message = "Internal server error")
     ))
   def dropMetric: Route =
-    pathPrefix("commands") {
-      pathPrefix(Segment) { db =>
-        pathPrefix(Segment) { namespace =>
-          pathPrefix(Segment) { metric =>
-            delete {
-              withMetricAuthorization(db, namespace, metric, true, authorizationProvider) {
-                onComplete(writeCoordinator ? DropMetric(db, namespace, metric)) {
-                  case Success(MetricDropped(_, _, _)) => complete("Ok")
-                  case Success(_)                      => complete(HttpResponse(InternalServerError, entity = "Unknown reason"))
-                  case Failure(ex)                     => complete(HttpResponse(InternalServerError, entity = ex.getMessage))
-                }
+    pathPrefix(Segment) { db =>
+      pathPrefix(Segment) { namespace =>
+        pathPrefix(Segment) { metric =>
+          delete {
+            withMetricAuthorization(db, namespace, metric, true, authorizationProvider) {
+              onComplete(writeCoordinator ? DropMetric(db, namespace, metric)) {
+                case Success(MetricDropped(_, _, _)) => complete("Ok")
+                case Success(_)                      => complete(HttpResponse(InternalServerError, entity = "Unknown reason"))
```

Comment:
does it make sense to you here to log in "entity" field also the object message type you have as the return of this success? Maybe helps debugging session.